AxonDeepSeg/integrity_test.py

```diff
@@ -26,6 +27,7 @@ def integrity_test():
         path_model = dir_path / 'models' / model_name
         path_testing = path_model / 'data_test'
         path_configfile = path_model / 'config_network.json'
+        image_name = "image.png"
```

Comment:
Make a `Path()` object so that you can fetch the stem in line 47 in the method I suggested above.

Reply:
Resolved in commit 90e53b5